modules/ROOT/pages/xml/xml-xpath.adoc

```diff
@@ -5,7 +5,7 @@ endif::[]
 
 The `<xml-module:xpath-extract>` operation supports the evaluation of XPath expressions.
 
-Because XPath expressions can match any number of individual elements, this operation returns a List of Strings. If no element matches the expression, the operation returns an empty list.
+Because XPath expressions can match any number of individual elements, this operation returns a list of strings. If no element matches the expression, the operation returns an empty list.
```

Comment:
Seems like it should be "list of Strings" -- to emphasize the use of the String class.